docs/prepare-feature.md

```diff
@@ -0,0 +1,70 @@
+#### prepare-feature
+---
+Generates `features.json` file for user feature(s). The `features.json` file is JSON file that contains the information found within a feature's ESA manifest file. JSONs are a key requirement for the installation of any liberty features(s) from a maven repository. 
+
+
+In Open Liberty and WebSphere Liberty runtime versions 21.0.0.11 and above, this goal can prepare the user feature to generate the JSON file in the following way:
+
+
+1. Create a `features-bom` file for the user feature. The `features-bom` artifact in each groupId provides the bill of materials (BOM) for each maven artifacts. 
+ ```xml
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>userTest.user.test.features</groupId>
+  <artifactId>features-bom</artifactId>
+  <version>21.0.0.11</version>
```

Comment:
I think using version `21.0.0.11` in all of these examples is confusing. These versions are in no way related to the Liberty version, correct? It is the version of the user feature.